### PR TITLE
CCM-5100: Update repo description for nhs-notify-iam-webauth

### PR DESCRIPTION
**Pull request by m-houston**
## Description

The `nhs-notify-iam` repo has been renamed to `nhs-notify-iam-webauth` and this PR updates the repo description accordingly

## Type of changes

- [ ] Refactoring (non-breaking change)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would change existing functionality)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] Documentation

## Checklist

- [x] I am familiar with the [contributing guidelines](../docs/CONTRIBUTING.md)
- [x] I have followed the code style of the project
- [ ] I have added tests to cover my changes
- [x] I have updated the documentation accordingly
- [ ] This PR is a result of pair or mob programming

---

## Sensitive Information Declaration

To ensure the utmost confidentiality and protect your and others privacy, we kindly ask you to NOT including [PII (Personal Identifiable Information) / PID (Personal Identifiable Data)](https://digital.nhs.uk/data-and-information/keeping-data-safe-and-benefitting-the-public) or any other sensitive data in this PR (Pull Request) and the codebase changes. We will remove any PR that do contain any sensitive information. We really appreciate your cooperation in this matter.

- [x] I confirm that neither PII/PID nor sensitive data are included in this PR and the codebase changes.

### CODE REVIEW

File: docs/_includes/page-info-header.html [hunk 1]
```diff
@@ -2,10 +2,10 @@
   assign wordWarning = 200 %} {% assign wordCount = content | number_of_words%} {%
   assign readTime = wordCount | divided_by: 100.0 | ceil %} {% if page.sub_title
   %}
-  <h1>{{ page.title }}</h1>
+  <h1>{{ page.name }}</h1>
```
Comment (m-houston) [inline docs/_includes/page-info-header.html:5]: I couldn't work out how to reconcile the 'name' and 'title' of a repos collection in a way that shows the actual repo slug in the nav menu. The auto-generated titles are confusing, so have overridden the page.title to be the repo slug and page.name to be a friendly name for the title in the actual page, but maybe this needs a different pattern to be clear.

File: docs/_layouts/notify-repo.md [hunk 2]
```diff
@@ -4,19 +4,15 @@ layout: page
 
 <!-- markdownlint-disable MD041 -->
 
-<p>
-<a href="https://github.com/NHSDigital/{{page.repo-name}}">https://github.com/NHSDigital/{{page.repo-name}}</a>
-</p>
+<dl>
```
Comment (m-houston) [inline docs/_layouts/notify-repo.md:7]: This is a more compact display of the repo frontmatter variables:
image

File: docs/_includes/page-info-header.html (hunk 1 above)
Comment (timireland) [reply docs/_includes/page-info-header.html:5]: The existing _repo collections md files will need

**Comment by m-houston** [comment]
Merging as a marginal improvement, will need revisiting once we decide how repo titles should be formatted

**Pull request merged**